README.md

```diff
@@ -57,6 +57,8 @@ it needs to be disabled.
 Download the archive matching your host OS from the Minishift [releases page](https://github.com/minishift/minishift/releases) and unpack it. Then copy the contained binary to your preferred
 location and optionally ensure it is added to your _PATH_.
 
+**Note**: For Windows OS, unpack the binary in your preferred location in the drive which contains your `%USERPROFILE%` directory. Mostly in `C:` drive.
+
```

Comment:
I don't understand why we need this `note` section. User can run binary for any location it's `%USERPROFILE%` suppose to be set as it's user home where it will create `.minishift` dir for everything.